remill/Arch/Runtime/HyperCall.h

```diff
@@ -87,6 +87,9 @@ class AsyncHyperCall {
     kX86SysEnter,
     kX86SysExit,
 
+    // Far jumps.
```

Comment:
Let's add a comment that the `CS` segment is updated before this hyper call.

A more general todo is to comment all hyper calls on their pre-conditions / what can be assumed.

Reply:
Updated.